deploy/examples/create-external-cluster-resources.py

```diff
@@ -1418,6 +1440,13 @@ def _gen_output_map(self):
                 self.out_map["CSI_CEPHFS_PROVISIONER_SECRET"],
                 self.out_map["CSI_CEPHFS_PROVISIONER_SECRET_NAME"],
             ) = self.create_cephCSIKeyring_user("client.csi-cephfs-provisioner")
+            self.get_or_create_subvolume_group(
+                self._arg_parser.subvolume_group,
+                self._arg_parser.cephfs_filesystem_name,
+            )
+            self.pin_subvolume(
+                "csi", self._arg_parser.cephfs_filesystem_name, "distributed", "1"
+            )
```

Comment:
You need to pin the subvolumegroup you have created, the script should create a default subvolumegroup called csi

Reply:
you should not pass `csi` here, you should pass `self._arg_parser.subvolume_group`

Reply:
I think calling the "csi" default directly would be fine,

I will only call if there is `filesystem name` and `CEPHFS_POOL_NAME` is present, so i guess that would be sufficient to understand if there is mds running or not, right?

Reply:
@Madhu-1 ^^